.travis.yml

```
@@ -49,6 +49,7 @@ jobs:
       script: ./vendor/bin/phpcs
 
     - stage: Code Quality
+      php: 7.4snapshot
```

A stable image is out
```suggestion
      php: 7.4
```